src/formatDate.js

```diff
@@ -50,7 +50,45 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateSeparator = '/';
+
+  if (date.includes('-')) {
+    dateSeparator = '-';
+  } else if (date.includes('.')) {
+    dateSeparator = '.';
+  }
+
+  const dateArray = date.split(dateSeparator);
+  const dateObject = {};
+  const result = [];
+
+  for (let i = 0; i < 3; i++) {
+    dateObject[fromFormat[i]] = dateArray[i];
+  }
+
+  if (fromFormat.includes('YY') && toFormat.includes('YYYY')) {
```

Comment:
first condition is redundant, because if fromFormat.includes('YYYY') that it's also included 'YY'
```suggestion
  if (toFormat.includes('YYYY')) {
```

Reply:
it doesn't work like that